pygbm/splitting.py

```diff
@@ -307,12 +311,12 @@ def _parallel_find_split_subtraction(context, parent_histograms,
     histograms by substraction.
     """
     # Pre-allocate the results datastructure to be able to use prange
-    split_infos = [SplitInfo(0, 0, 0, 0., 0., 0., 0.)
+    split_infos = [SplitInfo(0, 0, 0, 0., 0., 0., 0., 0, 0)
                    for i in range(context.n_features)]
```

Comment:
This data structure could probably be also stored as an attribute on the context to avoid reallocating it over and over again.